src/main/scala/org/broadinstitute/monster/extractors/xml/XmlExtractor.scala

```diff
@@ -29,8 +28,6 @@ class XmlExtractor private[xml] (blocker: Blocker)(implicit context: ContextShif
   /** Helper used to build XML readers when needed. */
   private val ReaderFactory = new WstxInputFactory()
 
-  private type Tag = Chain[XMLEvent]
```

Comment:
I originally tried to change this to be:
```scala
type Tag = NonEmptyChain[XmlEvent]
```

Unfortunately `NonEmptyChain` is also a `type` alias, and the double-hop was confusing IntelliJ. I figured the code brevity wasn't worth the false alarms of red squiggles in the IDE.